src/operator/bufferToggle.ts

```diff
@@ -44,18 +46,18 @@ import {errorObject} from '../util/errorObject';
  * @owner Observable
  */
 export function bufferToggle<T, O>(openings: Observable<O>,
```

Comment:
OK, will try to update. Would it be ok to accept `SubscribableOrPromise` as same as closing?